packages/now-build-utils/src/lambda.ts

```diff
@@ -105,3 +133,21 @@ export async function createZip(files: Files): Promise<Buffer> {
 
   return zipBuffer;
 }
+
+export async function getLambdaOptionsFromFunction(
+  sourceFile: string,
+  config?: Config
+): Promise<Pick<LambdaOptions, 'memory' | 'maxDuration'>> {
```

Comment:
Why does this return a promise? Are you thinking this might need to be async as some point?

Reply:
Possible, for example when it's needs to resolve files etc.